cmd/kpod/ps.go

```diff
@@ -299,6 +299,8 @@ func getTemplateOutput(containers []*libkpod.ContainerData, opts psOptions) (psO
 			status = "Exited (" + strconv.FormatInt(int64(ctr.State.ExitCode), 10) + ") " + runningFor + " ago"
 		case runningState:
 			status = "Up " + runningFor + " ago"
+		case "paused":
```

Comment:
You have a constant defined above for "running", and "stopped" and "paused".  Need to be consistant.  I would define them all as constants.

Reply:
But better yet, include oci/oci.go and get the constants from there.

Reply:
Fixed